src/storage/block_compactor.cpp

```diff
@@ -0,0 +1,360 @@
+#include "storage/block_compactor.h"
+#include <algorithm>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+#include "storage/index/bwtree_index.h"
+#include "storage/index/index_defs.h"
+#include "storage/sql_table.h"
+namespace terrier::storage {
+namespace {
+// for empty callback
+void NoOp(void * /* unused */) {}
```

Comment:
We've got an empty callback defined in transaction_util. Can we do something similar in storage_util?

Reply:
I can just use that one. I was under the impression we only had one in tests.